File: exercicios/Projeto Final/Projeto Back/Projeto/Model/Produto.cs
namespace Projeto.Model
{
    public class Produto
    {
        public int ID { get; set; }
        public String nome { get; set; }
        public double preco { get; set; }
        public bool ativo { get; set; }
        //public virtual ICollection<Venda> Vendas { get; set; }
    }
}

File: Projeto Angu/src/app/produto/produto.ts
export class Produto{
    id?: number;
    nome?: String;
    preco?: number;
    ativo?: boolean;

    constructor(nome?: String, preco?: number, id?: number){
        this.nome = nome;
        this.preco = preco;
        this.id = id;
        this.ativo = true;
    }
}